ci/create-multiarch-manifest.sh

```diff
@@ -0,0 +1,78 @@
+# Authenticate and retrieve DockerHub token
```

Comment:
I think this would benefit from more strictness.

```bash
#!/bin/bash

set -e -E -u -o pipefail
```

would you consider that?